services/src/main/java/org/keycloak/forms/login/freemarker/model/WebAuthnAuthenticatorsBean.java

```diff
@@ -34,8 +36,9 @@ public WebAuthnAuthenticatorsBean(KeycloakSession session, RealmModel realm, Use
                 .map(WebAuthnCredentialModel::createFromCredentialModel)
                 .map(webAuthnCredential -> {
                     String credentialId = Base64Url.encodeBase64ToBase64Url(webAuthnCredential.getWebAuthnCredentialData().getCredentialId());
-                    String label = (webAuthnCredential.getUserLabel()==null || webAuthnCredential.getUserLabel().isEmpty()) ? "label missing" : webAuthnCredential.getUserLabel();
-                    return new WebAuthnAuthenticatorBean(credentialId, label);
+                    String label = (webAuthnCredential.getUserLabel() == null || webAuthnCredential.getUserLabel().isEmpty()) ? "label missing" : webAuthnCredential.getUserLabel();
+                    String createdAt = DateTimeFormatterUtil.getDateTimeFromMillis(webAuthnCredential.getCreatedDate(), realm.getDefaultLocale());
```

Comment:
Question: will it be better if locale is determined as locale of current user instead of realm default locale?

It can be done as `session.getContext().resolveLocale(user)` . For example see some places in clas `FreemarkerLoginFormsProvider` how to determine locale of current user. The realm default locale can be possibly used just as a fallback if user is null (which AFAIK can happen in this bean, but I am not 100% sure. Also I am not 100% sure if this possibility of null user is already handled by the `session.getContext().resolveLocale(user)` and automatically handles fallback to realm default locale.

Reply:
Yep, it'll be better. I'll check it out.